components/rabbitmq/scripts/start.py

```diff
@@ -94,3 +95,25 @@ def set_rabbitmq_policy(name, expression, policy):
     # rabbitmq restart exits with 143 status code that is valid in this case.
     utils.start_service_and_archive_properties(RABBITMQ_SERVICE_NAME,
                                                ignore_restart_fail=True)
+    rabbitmq_endpoint_ip = '127.0.0.1'
+
+    if utils.systemd.is_alive(RABBITMQ_SERVICE_NAME):
+        ctx.logger.info('RabbitMQ service is running')
+    else:
+        ctx.logger.info('RabbitMQ service is not running')
+
+
+sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+try:
```

Comment:
another cross-script comment: please put everything in functions so that they can be easily unit tested